app/controllers/orders_controller.rb

```diff
@@ -1,10 +1,42 @@
 class OrdersController < ApplicationController
+
   def index
     @orders = current_user.orders
   end
 
+  def create
+    if current_user.nil?
+      flash[:login] = {color: 'red', message: "Please log in to check out}"}
+      redirect_to '/login'
+    else
+      @order = current_user.orders.create(
+        status: "ordered",
+        total_price: total_price
+      )
+      add_items_to_order
+      flash[:order_success] = {color: 'green', message: "Order was successfully placed}"}
```

Comment:
Line is too long. [89/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.